### PR TITLE
Update images configuration parameters

### PR DESCRIPTION
**Pull request by dmage**

### CODE REVIEW

File: modules/images-configuration-parameters.adoc [hunk 1]
```diff
@@ -5,67 +5,54 @@
...
 |`allowedRegistriesForImport`: Limits the container image registries from which
 normal users may import images. Set this list to the registries that you trust
 to contain valid images, and that you want applications to be able to
 import from. Users with permission to create images or `ImageStreamMappings`
 from the API are not affected by this policy. Typically only cluster
 administrators will have the appropriate permissions.
 
-`additionalTrustedCA`: A reference to a ConfigMap containing additional CAs that
+Every element of this list contains a location of the registry specified by the
+registry domain name.  The domain name might include wildcards.
```
Comment (bmcelvee) [inline modules/images-configuration-parameters.adoc:25]: s/might include/can include

**Comment by bmcelvee** [comment]
I just have one suggestion, otherwise LGTM. Since this is clarifying parameters, I don't think it requires QE review. Looks like it applies to 4.2+.

File: modules/images-configuration-parameters.adoc (hunk 1 above)
Comment (dmage) [reply modules/images-configuration-parameters.adoc]: updated

**Comment by bmcelvee** [comment]
Thanks, @dmage! I think this is ready to merge and cherry pick.

**Comment by mburke5678** [comment]
@dmage Sorry if this is an ignorant question: Should `registrySources` be in the table twice?

https://github.com/openshift/openshift-docs/pull/23201/files#diff-9d4142ac144208170b7c45a6164f31954dec2b2a0cbccb7611f19ffa8a5352feR43-R55

cc. @bmcelvee

**Comment by bmcelvee** [comment]
@mburke5678: strangely enough, yes: https://github.com/openshift/api/blob/275406142edb3c1f7a72f2cbeccfd2700f80b0a0/config/v1/types_image.go

Though we might be able to better clarify `ImageSpec` versus `RegistryLocation`.

**Pull request merged**